lighthouse-core/audits/no-unload-listeners.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Audit = require('./audit.js');
+const i18n = require('./../lib/i18n/i18n.js');
+
+const UIStrings = {
+  /** Descriptive title of a Lighthouse audit that checks if a web page has 'unload' event listeners and finds none. */
+  title: 'Avoids `unload` event listeners',
+  /** Descriptive title of a Lighthouse audit that checks if a web page has 'unload' event listeners and finds that it is using them. */
+  failureTitle: 'Listens for the `unload` event',
+  /** Description of a Lighthouse audit that tells the user why pages should not use the 'unload' event. This is displayed after a user expands the section to see more. 'Learn More' becomes link text to additional documentation. */
+  description: 'The `unload` event does not fire reliably and listening for it can prevent browser optimizations like the back/forward cache. Consider using the `pagehide` or `visibilitychange` events instead. [Learn More](https://developers.google.com/web/updates/2018/07/page-lifecycle-api#the-unload-event)',
```

Comment:
I think it's better to say "can prevent" since adding an unload listener won't prevent a page from being cached in Safari and Chrome (just Firefox).

Also, change "back/forward cache" to "Back-Forward Cache" and consider linking to: https://www.chromestatus.com/features/5815270035685376.

We should probably also update the link to point to something like `https://web.dev/back-forward-cache/`—assuming this won't launch publicly until after M86 hits beta?

Reply:
> Also, change "back/forward cache" to "Back-Forward Cache" and consider linking to: https://www.chromestatus.com/features/5815270035685376.

done

> We should probably also update the link to point to something like `https://web.dev/back-forward-cache/`—assuming this won't launch publicly until after M86 hits beta?

We can leave this with the working URL for now in case it ships earlier and then update quickly when that's ready.

Reply:
> > Also, change "back/forward cache" to "Back-Forward Cache" and consider linking to: https://www.chromestatus.com/features/5815270035685376.
> 
> done

Wellll, actually, it feels a little weird to link to a browser-specific docs page when it's already shipping in other browsers and the advice we're giving is for cross-browser bfcache compatibility. Let's just include Chrome's status in the future `https://web.dev/back-forward-cache/`?